config/routes.rb

```diff
@@ -3,6 +3,12 @@
     resources :results, only: [:new, :create]
   end
 
+  namespace :api do
+    resources :subjects, only: [], id: /[A-Z0-9\.]+?/i do
```

Comment:
The agreed upon identifier for a subject is its number, which looks something like 6.100A or 2.URG. Without overriding the id format here, Rails was not allowing the dot.